src/tests/components/mainView/topbar.spec.js

```diff
@@ -0,0 +1,77 @@
+define(['react', 'components/mainView/topbar'],
+    function (React, TopBar) {
+        'use strict';
+
+        var testUtils = React.addons.TestUtils;
+
+        describe('top bar', function () {
+            it('should show the logo of the app', function () {
+                var props = {
+                    username: 'username'
+                };
+                var topBarElement = React.createElement(TopBar, props);
+                var topBarComp = testUtils.renderIntoDocument(topBarElement);
```

Comment:
if the file is .jsx first three statements can become this:
topBarComp = TestUtils.renderIntoDocument(<TopBar username='username'/>);